ci/build_alpine_container/README.md

```diff
@@ -0,0 +1,17 @@
+This directory contains a Dockerfile that builds an envoy image
```

Comment:
OK, let's just push 2 images then on master commit. @junr03 can you review this to that effect.